### PR TITLE
fix: localAttributes and globalAttributes filtering and returned wrong values.

### PR DESCRIPTION
**Pull request by creative-andrew**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [X] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [X] Make sure you are requesting to pull request from a **topic/feature/bugfix/devops branch** (right side). Don't pull request from your master!
- [X] Have you ensured/updated that CLI tests to extend coverage to any new logic. Learn how to modify the tests [here](https://woographql.com/contributing/2-local-testing).

What does this implement/fix? Explain your changes.
---------------------------------------------------

- Removes `type` filtering on globalAttributes and localAtrributes
- Fixes `type` filtering on attributes
- Fixes correct data is returned for globalAttributes and localAttributes

Does this close any currently open issues?
------------------------------------------
Closes: https://github.com/wp-graphql/wp-graphql-woocommerce/issues/756

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
See: https://github.com/wp-graphql/wp-graphql-woocommerce/issues/756

### CODE REVIEW

File: includes/connection/class-product-attributes.php [hunk 1]
```diff
@@ -59,13 +65,20 @@ public static function register_connections() {
...
-
+					// phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+					switch ( $info->fieldName ) {
+						case 'globalAttributes':
+							$args['where']['type'] = 'global';
+							break;
+						case 'localAttributes':
+							$args['where']['type'] = 'local';
+							break;
+					}
```
Comment (kidunot89) [inline includes/connection/class-product-attributes.php:81]: Why not put this in the connection resolver's `get_query_args()` function?

File: includes/connection/class-product-attributes.php [hunk 2]
```diff
@@ -26,7 +26,13 @@ class Product_Attributes {
 	 */
 	public static function register_connections() {
 		// From Product to ProductAttribute.
-		register_graphql_connection( self::get_connection_config() );
+		register_graphql_connection(
+			self::get_connection_config(
+				[
+					'connectionArgs' => self::get_connection_args(),
+				]
```
Comment (kidunot89) [inline includes/connection/class-product-attributes.php:33]: This can be on one line

**Review by kidunot89 — CHANGES_REQUESTED** [review]
Looks good for the most part. Just see my comments

File: includes/connection/class-product-attributes.php (hunk 1 above)
Comment (creative-andrew) [reply includes/connection/class-product-attributes.php:79]: Yes. I will create a connection resolver for this on a different task, but for now, I think it's important to merge this PR as it fixes a critical error.

**Review by kidunot89 — APPROVED** [review]
Good one, bro👍🏿

**Review by kidunot89 — APPROVED** [review]
Good one, bro👍🏿

**Pull request merged**